core/src/main/scala/org/apache/spark/rdd/PairRDDFunctions.scala

```diff
@@ -131,7 +131,10 @@ class PairRDDFunctions[K, V](self: RDD[(K, V)])
     lazy val cachedSerializer = SparkEnv.get.serializer.newInstance()
     val createZero = () => cachedSerializer.deserialize[U](ByteBuffer.wrap(zeroArray))
 
-    combineByKey[U]((v: V) => seqOp(createZero(), v), seqOp, combOp, partitioner)
+    val cleanedSeqOp = self.context.clean(seqOp)
+    val cleanedCombOp = self.context.clean(combOp)
+    combineByKey[U](
+      (v: V) => cleanedSeqOp(createZero(), v), cleanedSeqOp, cleanedCombOp, partitioner)
```

Comment:
In places like `reduceByKey` we pass it directly to `combineByKey` so it's more natural. Here we do a bunch of stuff before we clean, and it's kinda weird to just clean one of passed in closures but not the other.

But this is a small enough point that I'll just skip the cleaning here.